Jenkinsfile

```diff
@@ -11,5 +11,7 @@ dockerfile {
     dockerImageClean = false
     downStreamRepos = ["confluent-security-plugins", "confluent-cloud-plugins"]
     nanoVersion = true
+    maxBuildsToKeep = 999
+    maxDaysToKeep = 90
```

Comment:
We might not need to set `maxDaysToKeep` since it is set default as 90 days. Consider Ksql logs is fairly big, "999" will used around 188GB, maybe "99" is more reasonable until we find a way to archive logs in other storage place.

Reply:
Thanks, @xli1996 !

I'll change it to 99 and we can re-evaluate to see if that's long enough (without using too much disk).

Regarding the retention time, I wanted to go ahead and make that default explicit so there's no mystery about what our retention time is, and it won't magically change on us if the defaults change.